packages/ckeditor5-bookmark/tests/bookmarkediting.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * @license Copyright (c) 2003-2024, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/* global document */
+
+import BookmarkEditing from '../src/bookmarkediting.js';
+
+import { Enter } from '@ckeditor/ckeditor5-enter';
+import { Paragraph } from '@ckeditor/ckeditor5-paragraph';
+import { Undo } from '@ckeditor/ckeditor5-undo';
+import { Link } from '@ckeditor/ckeditor5-link';
+
+import ClassicTestEditor from '@ckeditor/ckeditor5-core/tests/_utils/classictesteditor.js';
+
+describe( 'BookmarkEditing', () => {
+	let editor, element, model;
+
+	beforeEach( () => {
+		element = document.createElement( 'div' );
+		document.body.appendChild( element );
+
+		return ClassicTestEditor
+			.create( element, {
+				language: 'en',
+				plugins: [ BookmarkEditing, Enter, Paragraph, Undo, Link ]
+			} )
+			.then( newEditor => {
+				editor = newEditor;
+				model = editor.model;
+			} );
+	} );
+
+	afterEach( () => {
+		return editor.destroy().then( () => element.remove() );
+	} );
```

Comment:
I'd suggest using `async`/`await` as it's clearer.

Reply:
Addressed in 2e3d86b.